onnxruntime/test/contrib_ops/attention_op_test.cc

```diff
@@ -122,7 +124,7 @@ static void RunAttentionTest(
 }
 
 static void RunAttentionTest(
-    const std::vector<float>& input_data,         // input:      [batch_size, sequence_length, hidden_size]
+    const std::vector<float>& input_data,         // input:      [batch_size, sequence_length, hidden_size] or [sequence_length, batch_size, hidden_size]
```

Comment:
does the comment for output_data below need to be fixed up like above?

Reply:
oh yes!